src/WPF/ArcGISRuntime.WPF.StorePackage/ArcGISRuntime.WPF.StorePackage.wapproj

```diff
@@ -78,6 +78,8 @@
   <ItemGroup>
     <ProjectReference Include="..\ArcGISRuntime.WPF.Viewer\ArcGISRuntime.WPF.Viewer.Net.csproj">
       <SkipGetTargetFrameworkProperties>True</SkipGetTargetFrameworkProperties>
+      <PublishProfile Condition="'$(Configuration)|$(Platform)'=='Release|x64'">Properties\PublishProfiles\win-x64-release.pubxml</PublishProfile>
+      <!--<PublishProfile Condition="'$(Configuration)|$(Platform)'=='Release|x86'">Properties\PublishProfiles\win-x64-release.pubxml</PublishProfile>-->
```

Comment:
Should this be commented out? Unless it's one or other?

Reply:
This is intentional. To build the other package this needs to be uncommented.